From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Thu, 4 Apr 2024 17:53:09 -0600
Subject: [PATCH] kernelbase: HACK: Force CEF swiftshader for Insanity's Blade.

CW-Bug-Id: #23650
Valve-commit: 9d095387a73fe07c7852c10df21928b46832211d
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index 7b15b3f..9434a50 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -511,6 +511,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
     }
     options[] =
     {
+        {L"Insanitys Blade\\nw.exe", L" --use-gl=swiftshader"},
         {L"Warhammer2.exe", L" --in-process-gpu"},
         {L"SummerIslands.exe", L" --in-process-gpu"},
         {L"Paradox Launcher.exe", L" --use-angle=gl"},
